### ISSUE TITLE
Statistic fps comparison table

### ISSUE THREAD

**Issue by nvcken** [mathieuancelin/js-repaint-perfs#55]
A statistic  table to display avg fps would be awesome

**Comment by mathieuancelin** [comment]
Yep, that would actually be awesome, but we need something automatic, that will update the table for each merged PR. Like running all examples in a browser (with something like SauceLab or Browserstack) and extract new data to feed the table.

If you have any idea to easily achieve that, I'm open to it :-)

**Comment by leeoniya** [comment]
maybe avgs at 1%, 50%, 100% mutation rates
